modules/appeals_api/spec/support/rswag_config.rb

```diff
@@ -62,6 +62,14 @@ def config
         name: 'appealable_issues',
         tags: [{ name: 'Appealable Issues', description: '' }]
       ),
+      "modules/appeals_api/app/swagger/appeals_status/v1/swagger#{DocHelpers.doc_suffix}.json" => rswag_doc_config(
```

Comment:
This doc was separate from the decision reviews docs before any of the segmented API docs were split from decision reviews - we'd taken a different approach to handle it at the time, since it was the odd one out, but now that we're generating other docs besides just decision reviews and appeals status, it can have a real entry in this config hash